app/components/login_button_component.rb

```diff
@@ -0,0 +1,60 @@
+class LoginButtonComponent < BaseComponent
+  attr_reader :action, :big, :color, :width, :height, :logo_path, :tag_options
+
+  alias_method :big?, :big
+
+  DEFAULT_HEIGHT = 16
+  DEFAULT_WIDTH = 121
+  BIG_HEIGHT = 20
+  BIG_WIDTH = 152
+
+  def initialize(
+    action: ->(**tag_options, &block) { button_tag(**tag_options, &block) },
+    big: false,
+    color: "light blue",
```

Comment:
I wonder if it'd be better to align these to our color tokens (`primary-lighter`, etc.) to simplify the implementation, avoid inconsistency, and avoid hard-coding references to specific color in case of a future rebrand. If we wanted more human-readable labels like "light blue", I think we could implement that in the specific UI where the button is being used.